includes/Type/InterfaceType/EditorBlockInterface.php

```diff
@@ -42,109 +43,142 @@ public static function get_block( array $block ) {
+				'description'     => static fn () => __( 'Blocks that can be edited to create content and layouts', 'wp-graphql-content-blocks' ),
 				'fields'          => [
-					'clientId'                => [
-						'type'        => 'String',
-						'description' => __( 'The id of the Block', 'wp-graphql-content-blocks' ),
-						'resolve'     => static function ( $block ) {
-							return isset( $block['clientId'] ) ? $block['clientId'] : uniqid();
-						},
-					],
-					'parentClientId'          => [
-						'type'        => 'String',
-						'description' => __( 'The parent id of the Block', 'wp-graphql-content-blocks' ),
-						'resolve'     => static function ( $block ) {
-							return isset( $block['parentClientId'] ) ? $block['parentClientId'] : null;
-						},
-					],
-					'name'                    => [
-						'type'        => 'String',
-						'description' => __( 'The name of the Block', 'wp-graphql-content-blocks' ),
-					],
-					'blockEditorCategoryName' => [
-						'type'        => 'String',
-						'description' => __( 'The name of the category the Block belongs to', 'wp-graphql-content-blocks' ),
-						'resolve'     => static function ( $block ) {
-							return isset( self::get_block( $block )->category ) ? self::get_block( $block )->category : null;
-						},
-					],
-					'isDynamic'               => [
-						'type'        => [ 'non_null' => 'Boolean' ],
-						'description' => __( 'Whether the block is Dynamic (server rendered)', 'wp-graphql-content-blocks' ),
-						'resolve'     => static function ( $block ) {
-							return isset( self::get_block( $block )->render_callback );
-						},
-					],
-					'apiVersion'              => [
-						'type'        => 'Integer',
-						'description' => __( 'The API version of the Gutenberg Block', 'wp-graphql-content-blocks' ),
-						'resolve'     => static function ( $block ) {
-							return isset( self::get_block( $block )->api_version ) && absint( self::get_block( $block )->api_version ) ? absint( self::get_block( $block )->api_version ) : 2;
-						},
-					],
-					'innerBlocks'             => [
-						'type'        => [
-							'list_of' => 'EditorBlock',
-						],
-						'description' => __( 'The inner blocks of the Block', 'wp-graphql-content-blocks' ),
-						'resolve'     => static function ( $block ) {
-							return isset( $block['innerBlocks'] ) && is_array( $block['innerBlocks'] ) ? $block['innerBlocks'] : [];
-						},
-					],
-					'cssClassNames'           => [
-						'type'        => [ 'list_of' => 'String' ],
-						'description' => __( 'CSS Classnames to apply to the block', 'wp-graphql-content-blocks' ),
-						'resolve'     => static function ( $block ) {
-							if ( isset( $block['attrs']['className'] ) ) {
-								return explode( ' ', $block['attrs']['className'] );
-							}
+					// @TODO - Remove when WPGraphQL min version is 2.3.0
+					'clientId'                => WPGraphQLRegisterConfig::resolve_graphql_config(
```

Comment:
You over compensated vs the previous change. resolve_graphql_config, takes an _entire_ config object, so _if_ `Config.fields` is set, it will be handled. The previous problem was that `register_graphql_fields()` doesnt take a single `Config` (for any type), it takes an _array_ of `Configs`. You'll see it in my incoming PR, just explaining in context.